### PR TITLE
Fix macOS warning message 

### PR DESCRIPTION
**Pull request by jfedak**
## Description

This PR just fix a message in native view handler warning. 

## Test plan

n/a

### CODE REVIEW

File: apple/Handlers/RNNativeViewHandler.mm [hunk 1]
```diff
@@ -216,7 +216,7 @@ @implementation RNNativeViewGestureHandler
 
 - (instancetype)initWithTag:(NSNumber *)tag
 {
-  RCTLogWarn(@"Manual gesture handler is not supported on macOS");
+  RCTLogWarn(@"Native view gesture handler is not supported on macOS");
```
Comment (m-bert) [inline apple/Handlers/RNNativeViewHandler.mm:219]: I think this looks better. Let me know what you think.

```suggestion
  RCTLogWarn(@"NativeViewGestureHandler is not supported on macOS");
```

**Review by m-bert — APPROVED** [review]
Good catch. Also, I think that using convention like in suggestion below looks a bit better, but since all other warnings use spaces, I'll leave it up to you to decide.

File: apple/Handlers/RNNativeViewHandler.mm (hunk 1 above)
Comment (jfedak) [reply apple/Handlers/RNNativeViewHandler.mm:219]: Yup, imo it looks better for long names. When I wrote code for warnings for the first time I followed the message from hover handler on tvOS, but now I see that we do different for force touch (one single word).

Comment (j-piasecki) [reply apple/Handlers/RNNativeViewHandler.mm:219]: Yeah, it looks nicer. I'm fine with both, merging this one and changing them all in a follow-up, or just changing them in this PR.

**Review by j-piasecki — APPROVED** [review]

**Review by m-bert — APPROVED** [review]

**Pull request merged**